pkg/backup/backuper.go

```diff
@@ -247,6 +247,7 @@ func open(uri string) (DB, error) {
 	if err != nil {
 		return nil, errors.Errorf("opening db: %s", err)
 	}
+	db.SetMaxIdleConns(0)
```

Comment:
Undoing https://github.com/tablelandnetwork/go-tableland/pull/328/files#diff-c95ef053206f27ba2307970f3b1bfaa55fcd806df5e69e44123c010ccf796ac7L250 since this leaves `-(shm-wal)` files in the backup folder.